Automattic/pocket-casts-android
# End of Year: Show longest episode played

| 📘 Project: #410 |
|:---:|

iOS PR: https://github.com/Automattic/pocket-casts-ios/pull/416

## Description
Shows the longest episode played story.

## Testing Instructions
1. Set `END_OF_YEAR_ENABLED` feature flag to `true` in `base.gradle`
2. Run the app and make sure you're logged in to an account with a few items on Listening History
3. When the prompt appears, tap "View My 2022"
4. Skip to the last story.
5. ✅ Check that the longest episode that was played is shown, alongside its duration and podcast thumb

## Screenshots or Screencast 

<img src="https://user-images.githubusercontent.com/1405144/199906967-b1b84ceb-4f87-4d29-a036-6f10d75fc1f5.png" width="300">

## Checklist
 
#### I have tested any UI changes...
